oop/src/helpers/date.js

```diff
@@ -0,0 +1,8 @@
+import { format } from 'date-fns';
+
+export const formatDate = (date) => {
+  if (!date) {
+    return null;
+  }
+  return format(date, 'MMMM d, y')
+};
```

Comment:
Is it possible to put this on DateInput.vue since it's also importing from 'date-fns' or should it be a separate file?

Reply:
This is a helper method, so I think it's best to keep it separate from the `DateInput` component.